File: FouFouFood_ServeurNodeJS/foufoufood_serveur/MenuItem.js
class MenuItem {
    constructor(id, name, description, price, category, image, restaurantId) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.price = price;
        this.category = category;
        this.image = image;
        this.restaurantId = restaurantId;
    }
}

module.exports = MenuItem;

File: foufoufood_client_android/app/src/main/java/com/example/foufoufood/model/MenuItem.kt
package com.example.foufoufood.model

import com.google.gson.annotations.SerializedName

data class MenuItem(
    @SerializedName("_id") val id: String,
    val name: String,
    val description: String,
    var price: Float,
    val category: String,
    var image: String,
    val restaurantId: String
)
